### DISCUSSION TITLE
Question: Is Week 6 supposed to complain about missing assets? Are the tracklists for each week supposed to cut off the last song?

### DISCUSSION THREAD

**Discussion by MartianBinonwangan** [FunkinCrew/Funkin#1559] [Q&A]
## What is your question?

I was doing a general run-through of the game after I finished adding in a new level with dialogue into the game (I'm using the source code version btw) (I also would like to state that I only added one level and created a new week. Both run as intended). However, whenever I get to Week 6, it gives me multiple warnings when I first start up the song:

[WARNING] No animation called "greenScroll"
[WARNING] No animation called "blueScroll"
[WARNING] No animation called "redScroll"
[WARNING] No animation called "purpleScroll"
[WARNING] No animation called "normal"

While it does give these warnings, the animation and level seem to play fine. But still, I want to know if this is supposed to happen. I've used the desktop/downloadable version of the game and the console never popped up with these warnings.

Secondly, I noticed that in the campaign menu each of the weeks cuts off the last song in its list. For example, if week 6 has senpai, roses, and thorns, it will only show senpai and roses under the Tracks section (For tutorial it's just blank). However, whenever I play the weeks through the story mode menu, the weeks still play all the songs even though it always cuts off one. Is this normal?

BTW, I would also like to mention that I did redownload the package and recompile it. Both issues still persist even without any modified code. I would also like to mention that I'm just starting my modding adventure in FNF. While I do understand Java pretty well, the coding environment in FNF is really new to me so I'm still adapting.

**Comment by ghost** [comment]
I recommend downloading git-scm and using git clone
OR
clone with GitHub Desktop
